K8s: add real words to example values (#2606)
RedisLabs/redislabs-docs: content/kubernetes/active-active/prepare-clusters.md

> The combination of the REC name and namespace (`<rec-name>.<namespace-name>`) must be unique for each participating cluster the Active-Active database.

Reviewer: Missing word?
```suggestion
The combination of the REC name and namespace (`<rec-name>.<namespace-name>`) must be unique for each participating cluster in the Active-Active database.
```

> For example, if you have two K8s clusters, each with their own REC named `rec-chicago` in a namespace named `ns-illinois`. The value of `<rec-name>.<namespace-name>` for both RECs would be `rec-chicago.ns-illinois`. These can't be used for the same Active-Active database.

Reviewer: Sentence fragment fix
```suggestion
For example, if you have two K8s clusters, each with their own REC named `rec-chicago` in a namespace named `ns-illinois`, the value of `<rec-name>.<namespace-name>` for both RECs would be `rec-chicago.ns-illinois`. These can't be used for the same Active-Active database.
```

> The admission controller using a validating webhook to dynamically validate resources configured by the operator. The `ValidatingWebhookConfiguration` is required for Active-Active databases. Learn how to enable and configure admission controller in the [Enable admission controller]({{<relref "/kubernetes/deployment/quick-start.md#enable-the-admission-controller/">}}) section of the [Deploy Redis Enterprise Software for Kubernetes]({{<relref "/kubernetes/deployment/quick-start.md">}}) instructions.

Reviewer: ```suggestion
The admission controller uses a webhook to dynamically validate resources configured by the operator. The `ValidatingWebhookConfiguration` is required for Active-Active databases. Learn how to enable and configure the admission controller in the [Enable admission controller]({{<relref "/kubernetes/deployment/quick-start.md#enable-the-admission-controller/">}}) section of the [Deploy Redis Enterprise Software for Kubernetes]({{<relref "/kubernetes/deployment/quick-start.md">}}) instructions.
```
